test/spec/requestBuilder.js

```diff
@@ -16,3 +44,16 @@ test('The request builder should generate queries according to the search parame
 
   t.end();
 });
+
+test('The request builder should should force clickAnalytics to false on subsequent queries if not specified', function(t) {
+  var testData = require('./search.testdata.js')();
+  var searchParams = testData.searchParams;
+
+  var queries = getQueries(searchParams.index, searchParams);
+  t.equal(queries.length, 2);
+  t.equal(queries[0].params.clickAnalytics, undefined, 'the parameter clickAnalytics should be defined on the first query');
```

Comment:
Why we leave the value as `undefined` instead of explicitly set to `true` or `false`?

Reply:
From the Algolia rest API perspective there is already a default value set to `false`. We force the value to `false` on subsequent because we know for sure that those queries should never be analyzed. On the first query, it's less clear so we leave the engine decides what behavior it should use.